IPython/core/magics/execution.py

```diff
@@ -85,14 +85,20 @@ def stdev(self):
         return (math.fsum([(x - mean) ** 2 for x in self.timings]) / len(self.timings)) ** 0.5
 
     def __str__(self):
-        return (u"%s loop%s, average of %d: %s +- %s per loop (using standard deviation)"
-                   % (self.loops,"" if self.loops == 1 else "s", self.repeat,
-                      _format_time(self.average, self._precision),
-                      _format_time(self.stdev, self._precision)))
+        return (
+            u"{mean} ± {std} per loop (mean ± std. dev. of {runs} run{run_plural}, {loops} loop{loop_plural} each)"
```

Comment:
Are we comfortable assuming that ± will render fine everywhere? I recall looking at this ages ago and deciding for the more pedestrian/ugly +- precisely because ± didn't render very reliably.